recipes/qtxlsxwriter/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(${PROJECT_NAME} example.cpp)
+
+# workaround to deal with the error:
+# "Qt requires a C++11 compiler and yours does not seem to be that."
+set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 11)
+
+target_include_directories(${PROJECT_NAME} PRIVATE ${CONAN_INCLUDE_DIRS})
+target_link_directories(${PROJECT_NAME} PRIVATE ${CONAN_LIB_DIRS})
```

Comment:
not required when linking to targets

```suggestion
```

Reply:
In this test package I can't link to the target generated by the CMake generator because I need to set the order of libraries manually and list `Qt5Gui` first, since the current qt recipe provides an incorrect order of libraries (`qt::core` should go after `qt::gui`). For this reason I have to use the variables provided by  the generator in `target_include_directories` and `target_link_directories`.